### PR TITLE
Complete basic page layout setup 

### PR DESCRIPTION
**Pull request by velezjose**
The following components are now displayed with nicer CSS styling:
- Image icon list to the left of the page
- Image last hovered over as the main image (and its icon on the left surrounded by orange border and shadow)
- Title, Seller, Reviews, Price, Prime Eligible, Description

Moreover, all inline styles were refactored to use Styled Components. Code looks much cleaner.

### CODE REVIEW

**Review by alexminer88 — COMMENTED** [review]
This might be just a stylistic choice, but why did you decide to include the styles within the body of each component as opposed to a separate styles.css? I was under the impression that this was preferred (Seiji's conversation with Maia).

File: client/src/components/DescriptionView.jsx [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+// const primeURL = 
+
+const Description = styled.div`
+  font-size: 13px;
+`;
 const Paragraph = ({ paragraph }) => (<p>{ paragraph }</p>);
 
-const DescriptionView = ({ description }) => (
-  <div style={{fontSize: 13}}>
+const DescriptionView = ({ price, prime, description}) => (
```
Comment (alexminer88) [inline client/src/components/DescriptionView.jsx:13]: I noted that in most of your components, you deconstruct the incoming props object into variables. I quite like this approach, it feels much less convoluted than referring to the properties of a props object.

File: client/src/components/PicturesView.jsx [hunk 2]
```diff
@@ -23,15 +24,11 @@ class PicturesView extends React.Component {
...
-            pictureURLs={this.state.pictures} 
-            onHover={this.onIconHoverHandler.bind(this)}
-            idSelected={this.state.idSelected} />
-        </div>
-        <div>
-          <PictureDisplay pictureURL={this.state.current} numPictures={this.state.pictures.length} />
-        </div>
+        <PictureList 
+          pictureURLs={this.state.pictures} 
+          onHover={this.onIconHoverHandler.bind(this)}
```
Comment (alexminer88) [inline client/src/components/PicturesView.jsx]: As you mentioned in my review, you might want to put the bindings within the constructor, so the function that you pass looks cleaner

File: client/src/components/Product.jsx [hunk 3]
```diff
@@ -46,33 +44,34 @@ class Product extends React.Component {
...
+        <ContentSection>
+          <TitleView title={this.state.name} seller={this.state.seller} />
+          <ReviewsView
+            one={this.state.one_star_reviews}
+            two={this.state.two_star_reviews}
+            three={this.state.three_star_reviews}
+            four={this.state.four_star_reviews}
+            five={this.state.five_star_reviews}
+            total={this.state.total_reviews}
+            answered_questions={this.state.answered_questions} />
```
Comment (alexminer88) [inline client/src/components/Product.jsx:68]: A very small stylistic choice. Do you think it would be appropriate to move the /> to a new line?

**Review by alexminer88 — APPROVED** [review]

File: client/src/components/DescriptionView.jsx (hunk 1 above)
Comment (velezjose) [reply client/src/components/DescriptionView.jsx:13]: Yeah it looks nicer too!

File: client/src/components/Product.jsx (hunk 3 above)
Comment (velezjose) [reply client/src/components/Product.jsx:68]: I don't really know. I thought the same thing myself when writing this line of code and decided not to for no particular reason.

File: client/src/components/PicturesView.jsx (hunk 2 above)
Comment (velezjose) [reply client/src/components/PicturesView.jsx]: Yep, you're right.

**Comment by velezjose** [comment]
Regarding your question of whether to include styles in the body of the component or not:

- I don't really remember what Seiji and Maia ended up concluding, but I do remember Maia suggesting for us to use Styled Components instead of CSS Modules, so I did this. I don't really know the preferences of how/where to write the Styled Components, so you might be right and I might have to refactor these components later on to be in their own CSS files.

**Pull request merged**